packages/headless/src/controllers/commerce/product-view/headless-product-view.ssr.ts

```diff
@@ -10,7 +10,7 @@ import {
 } from './headless-product-view';
 
 export interface ProductViewDefinition
-  extends SharedControllerDefinitionWithoutProps<ProductView> {}
+  extends UniversalControllerDefinitionWithoutProps<ProductView> {}
```

Comment:
It would be pretty odd to use this controller on a search or listing page: its only use is to log a product view event when a PDP is loaded. Normally the PDP would either contain recommendations carousels or no Coveo-driven components at all aside from the standalone search box and the product view controller.